src/Components/VoteArticle.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+import { useParams } from "react-router-dom";
+import { patchVoteByArticleId } from "../api";
+import "../CSS/VoteArticle.css";
+
+function VoteArticle(props) {
+  const { articleVotes } = props;
+  const { article_id } = useParams();
+  const [votes, setVotes] = useState(articleVotes);
```

Comment:
Please add a state and handle errors, for example when trying to vote but the app is offline.